doc/manual/workflow-tips.rst

```diff
@@ -75,7 +75,7 @@ issuing the command::
 
 If you further add the following to your ``.juliarc.jl`` file ::
 
-    isinteractive() && isfile("_init.jl") && require("_init.jl")
+    isfile("_init.jl") && include(joinpath(pwd(), "_init.jl"))
```

Comment:
The example on the docs puts the case when `_init.jl` is in the current dir:
`...then calling julia from that directory...`
I figured `isfile` checks current dir, but `include` will check the dir where `.juliarc.jl` is located.